R/generateEDGEdata.R

```diff
@@ -25,7 +25,7 @@
 generateEDGEdata <- function(input_folder, output_folder,
                              EDGE_scenario, REMIND_scenario="SSP2",
                              IEAbal=NULL, GDP_country=NULL, RatioPPP2MER_country = NULL, POP_country=NULL, JRC_IDEES_Trsp=NULL, JRC_IDEES_MarBunk=NULL, trsp_incent=NULL,
-                             saveRDS=FALSE){
+                             storeRDS=FALSE){
```

Comment:
should also change here https://github.com/pik-piam/mrremind/blob/2303fb4a0a456c7b31e80ca878bb25fbfbb8ba62/R/calcEDGETrData.R#L33